packages/graphql-yoga/src/types.ts

```diff
@@ -39,10 +39,19 @@ export type CORSOptions =
     }
   | false;
 
-declare global {
-  interface ReadableStream<R = any> {
-    [Symbol.asyncIterator]: () => AsyncIterator<R>;
-  }
+/**
+ * Overrides the standard library definition to include AsyncIterator support.
+ *
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AsyncIterator
+ */
+export interface YogaReadableStreamOverride<R = any> extends ReadableStream<R> {
```

Comment:
I was a bit torn on this name. I was thinking of using `ReadableStreamWithAsyncIterator`.
Would love some more perspectives on this name